<launch>
    <node pkg="panda_gazebo" type="panda_moveit_server.py" name="panda_moveit_planner_server" required="true">
        <param name="create_extra_services" value="true" />
    </node>
</launch>